### PR TITLE
GVA Response Text UI

### PR DESCRIPTION
**Pull request by rasmustautsglia**
This PR creates GVA Response text UI. This PR also splits `ChatView.swift` a little bit, since it was closing on 1000 lines of code which is not desired and hard to read. The main file to review is `GvaResponseTextView.swift`. In addition, a typo in Gallery Card parsing was found and removed.

Testing will be done in a separate ticket.

MOB-2363
MOB-2366

### CODE REVIEW

File: GliaWidgets/Sources/View/Chat/Message/GvaResponseTextView.swift [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import UIKit
+
+class GvaResponseTextView: ChatMessageView {
```
Comment (EgorovEI) [inline GliaWidgets/Sources/View/Chat/Message/GvaResponseTextView.swift:3]:
```suggestion
final class GvaResponseTextView: ChatMessageView {
```

File: GliaWidgets/Sources/View/Chat/Message/GvaResponseTextView.swift [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            } else {
+                operatorImageView?.removeFromSuperview()
+                operatorImageView = nil
+            }
+        }
+    }
+
+    private let viewStyle: OperatorChatMessageStyle
+    private var operatorImageView: UserImageView?
+    private var operatorImageViewContainer = UIView().makeView()
```
Comment (EgorovEI) [inline GliaWidgets/Sources/View/Chat/Message/GvaResponseTextView.swift:29]: Could it be `let`?

**Review by EgorovEI — APPROVED** [review]

File: GliaWidgets/Sources/View/Chat/Message/GvaResponseTextView.swift (hunk 1 above)
Comment (rasmustautsglia) [reply GliaWidgets/Sources/View/Chat/Message/GvaResponseTextView.swift]: fixed [114977d](https://github.com/salemove/ios-sdk-widgets/pull/673/commits/114977de071802ccf684ae14daa3ea299a7b842c)

File: GliaWidgets/Sources/View/Chat/Message/GvaResponseTextView.swift (hunk 2 above)
Comment (rasmustautsglia) [reply GliaWidgets/Sources/View/Chat/Message/GvaResponseTextView.swift]: fixed [114977d](https://github.com/salemove/ios-sdk-widgets/pull/673/commits/114977de071802ccf684ae14daa3ea299a7b842c)

**Review by gersonnoboa — APPROVED** [review]

**Comment by rasmustautsglia** [comment]
!squash

**Review by igorkravchenko — APPROVED** [review]

**Pull request merged**